### PR TITLE
JR2-1177 Show warn messages on home page

### PR DESCRIPTION
**Pull request by varusakov**

### CODE REVIEW

File: src/main/webapp/app/home/status-notification/status-notification.component.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    constructor(private activeRouter: ActivatedRoute,
+                private store: Store<HomeState>) {
+    }
+
+    ngOnInit(): void {
+        this.showMessage$ = this.store.select(getStatusNotificationShowMessage);
+        this.store.select(getStatusNotificationMessageKey)
+            .takeUntil(this.unsubscribe$)
+            .subscribe((messageKey) => {
+                this.title = `home.${messageKey}.title`;
```
Comment (pavax) [inline src/main/webapp/app/home/status-notification/status-notification.component.ts]: if a translation is not available please do not show the message

File: src/main/webapp/i18n/translations.csv [hunk 2]
```diff
@@ -657,6 +657,10 @@ home,home.registration.url,https://www.arbeit.swiss/secoalv/de/home/menue/untern
 home,home.title,Job-Room,Job-Room,Job-Room,Job-Room
 home,home.title.portal,arbeit,work,travail,lavoro
 home,home.title.portal-url,https://www.arbeit.swiss,https://www.arbeit.swiss/secoalv/en/home.html,https://www.arbeit.swiss/secoalv/fr/home.html,https://www.arbeit.swiss/secoalv/it/home.html
+home,home.login-cancel.title,Login failed,Login failed,Login failed,Login failed
```
Comment (pavax) [inline src/main/webapp/i18n/translations.csv]: login-cancel.title:
```
DE: Login abgebrochen
FR: xxxxxxxxxxxxxxxxxxxxxx
EN: Login canceled
IT: xxxxxxxxxxxxxx
```

File: src/main/webapp/i18n/translations.csv [hunk 3]
```diff
@@ -657,6 +657,10 @@ home,home.registration.url,https://www.arbeit.swiss/secoalv/de/home/menue/untern
...
+home,home.login-cancel.message,You have canceled the login,You have canceled the login,You have canceled the login,You have canceled the login
```
Comment (pavax) [inline src/main/webapp/i18n/translations.csv]: login-cancel.message
`
DE: Sie haben den Login Vorgang abgebrochen
FR: XXXXXXXXXXXXXXXXXXX
EN: You have canceled the login
IT: XXXXXXXXXXXXXXXXXXX
`

File: src/main/webapp/app/home/status-notification/status-notification.component.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            })
+            .flatMap((state) => this.translateService.get([this.title, this.message])
+                .map((translations) => state.showMessage
+                    && this.hasTranslation(translations, this.title)
+                    && this.hasTranslation(translations, this.message))
+            );
+    }
+
+    private hasTranslation(translations: any, key: string): boolean {
+        return translations[key] && translations[key].lastIndexOf('translation-not-found', 0) < 0;
```
Comment (pavax) [inline src/main/webapp/app/home/status-notification/status-notification.component.ts]: this is kinda ugly

Comment (varusakov) [reply src/main/webapp/app/home/status-notification/status-notification.component.ts]: @pavax no, I didn't find better way. We can only change value returned by translation service for missing translation as describe here https://github.com/ngx-translate/core#how-to-handle-missing-translations

**Pull request merged**